batchOperations/batch_operations_create_test.js

```diff
@@ -39,7 +39,9 @@ describe('Batch Entity Creation. JSON', () => {
 
         const response = await http.post(batchCreationResource, entities);
 
-        expect(response.response).toHaveProperty('statusCode', 200);
-        assertBatchOperation(response, [entities[0].id, entities[1].id], []);
+        expect(response.response).toHaveProperty('statusCode', 201);
```

Comment:
201 ...
Didn't we say that batch ops should give either 204 or 207 ?

Reply:
Ah, it's create ... OK, create may give 201 or 207, while update/upsert give 204 or 207 ...
This would need to be specified in the NGSi-LD spec, of course ...

Reply:
NTC